packages/grafana-runtime/src/components/NestedFolderPicker.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+enum PermissionLevelString {
```

Comment:
This one should probably also be exported since it is part of the `NestedFolderPickerProps`.